compairifuel/lib/src/global/extension/provider.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:async';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+extension CacheForExtension on Ref {
+  void cacheFor(Duration duration) {
+    Timer? existingTimer = read(_cacheForTimerProvider);
+    if (existingTimer != null && existingTimer.isActive) {
+      print("A timer already exists for this provider. ${toString()}");
```

Comment:
The info prints make the build fail.
   info • Don't invoke 'print' in production code • lib/src/global/extension/provider.dart:8:7 • avoid_print
   info • Don't invoke 'print' in production code • lib/src/global/extension/provider.dart:15:7 • avoid_print
   info • Don't invoke 'print' in production code • lib/src/global/extension/provider.dart:21:7 • avoid_print
   info • Don't invoke 'print' in production code • lib/src/global/extension/provider.dart:26:7 • avoid_print
   info • Don't invoke 'print' in production code • lib/src/global/extension/provider.dart:32:7 • avoid_print